### DISCUSSION TITLE
救救孩子

### DISCUSSION THREAD

**Discussion by zoushengwu** [ikas-mc/ContextMenuForWindows11#177] [General]
老哥,我编译ContextMenuCustomHost老是过不了,两个依赖库也安装了
![image](https://github.com/user-attachments/assets/5b74958e-ffdb-4a3d-8d7b-8f111817ffea)
使用的是VS2022,
![image](https://github.com/user-attachments/assets/48c8cae4-22f2-47fa-937b-ed9bcfb2093e)

**Comment by ikas-mc** [comment]
像是环境问题,你试试vs新建一个cpp/winrt项目看看可以编译吗

![image](https://github.com/user-attachments/assets/db2513c2-606a-431c-aa4b-0a42db2ebe9a)

**Reply by zoushengwu** [reply]
感谢老哥,我把nugut缓存清理了一下就行了
